data.sql

```diff
@@ -1,5 +1,34 @@
 /* Populate database with sample data. */
-INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (1, 'Agumon', '2020-02-03', 0, true, 10.23);
- INSERT INTO animals VALUES (2, 'Gabumon', '2028-11-15', 2, true,8);
-INSERT INTO animals VALUES (3, 'Pikachu', '2021-01-07',1,false,15.04);
- INSERT INTO animals VALUES (4, 'Devimon', '2017-03-12',5,false,11);
\ No newline at end of file
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES (1, 'Agumon', '2020-02-03', 0, true, 10.23);
+
+INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES (2, 'Gabumon', '2028-11-15', 2, true,8);
+
+INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES (3, 'Pikachu', '2021-01-07',1,false,15.04);
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES (4, 'Devimon', '2017-03-12',5,false,11);
+
+INSERT INTO animals ( id,name, date_of_birth, escape_attempts, neutered, weight_kg, species) 
+VALUES (5, 'Charmander','2020-02-08',0,false,-11,'unknown');
+
+INSERT INTO animals 
+VALUES (6, 'Plantmon', '2021-11-15', 2, true, -5.7, 'unknown');
+
+INSERT INTO animals 
+VALUES (7, 'Squirtle', '1993-04-02', 3, false, -12.13, 'unknown');
+
+INSERT INTO animals 
+VALUES (8, 'Angemon', '2005-06-12', 1, true, -45, 'unknown');
+
+INSERT INTO animals 
+VALUES (9,'Boarmon', '2005-06-07', 7, true, 20.4, 'unknown');
+
+INSERT INTO animals 
+VALUES (10,'Blossom', '1998-10-13', 3, true, 17, 'unknown');
+
+INSERT INTO animals 
+VALUES (11,'Ditto', '2022-05-14', 4, true, 22, 'unknown');
```

Comment:
As required by the previous reviewer, I also noticed that your code has two extra columns: id and species that are not instructed.  The instructions only asked for name, date_of_birth, escape_attempts, neutered, and weight_kg columns. Here is a suggestion to improve your code:

- Remove the id column from the INSERT statements. The id column is an auto-incrementing primary key that is automatically generated by the database when a new row is inserted. Therefore, there is no need to specify the id values manually. This will also avoid any potential conflicts or errors if the id values are not unique or sequential.

- Please remove the species column from your code and resubmit.


Please make the required change and resubmit your code for another review. If you have any questions or need any help, feel free to reach out to me. Thank you for your hard work and cooperation.